modmesh/app/euler1d.py

```diff
@@ -692,13 +842,27 @@ def content(self):
                     ComboBoxItem("Euler1D-CESE")
                 Label("Configuration")
                 with Scroll():
-                    Table(self.config)
+                    Table(self.solver_config)
                 Button("Set").click(self.app.set)
+                Button("Option").click(self.on_open)
             with VBox().layout(weight=1):
                 Spacer()
                 Button("Start").click(self.app.start)
                 Button("Stop").click(self.app.stop)
                 Button("Step").click(self.app.step)
+            with (Modal(self.state("plot_config_open"),
+                        title="Plot configuration")
+                  .open(self.on_open)
+                  .close(self.on_close)):
+                with VBox():
+                    with HBox():
+                        Label("Layout selection")
+                        Button("Grid").click(lambda: self.app.grid_layout())
+                        Button("Single").click((lambda:
+                                                self.app.single_layout()))
+                    Label("Data line configuration")
+                    Table(self.plot_config)
+                    Button("Save").click(lambda: self.app.update_layout())
```

Comment:
Add a modal window that allows the user to select which data line is plotted and adjust the upper/bottom limit of the y-axis. Additionally, it should be able to switch the plot layout.